.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     # Install JS dependencies.
     - run:
         working_directory: shared
-        command: yarn install --pure-lockfile --ignore-engines --prefer-offline
```

Comment:
workaround to https://github.com/yarnpkg/yarn/issues/2629

Reply:
Can you reference the issue in a comment?

Reply:
what do you mean?

Reply:
can you add a comment linking to https://github.com/yarnpkg/yarn/issues/2629 as a comment in the file itself